Troubleshooting.md

```diff
@@ -184,6 +184,27 @@ To github.com:{{GITHUB USERNAME}}/oppia.git
 
 Note that the output above will look somewhat different for you since it's specific to what data you're pushing.
 
+### Exception: Error compiling proto files
+
+If you are getting something like this in M1 macbooks when running `python -m scripts.start`:
+
+```
+b'protoc-gen-js: program not found or is not executable\Please specify a program using absolute path or make sure the program is available in your PATH system variable\n--js_out: protoc-gen-js: Plugin failed with status code 1. Inplugin js: exit status 1\n
+Traceback (most recent call last) :
+   File "/Users/hasitha/.pyenv/versions/3.7.10/lib/python3.7/runpy.py", line 193, in _run_module_as_main
```

Comment:
You might want to change "hasitha" to "......" here, see other examples on this page.